server.js

```diff
@@ -1,42 +1,91 @@
+let mysql = require("mysql");
+let config = require("./config.js");
+const fetch = require("node-fetch");
 const express = require("express");
 const path = require("path");
 const bodyParser = require("body-parser");
 
-const { response } = require('express');
+const { response } = require("express");
 const app = express();
 const port = process.env.PORT || 5000;
-app.use(bodyParser.json({ limit: '50mb' }));
-app.use(bodyParser.urlencoded({ limit: '50mb', extended: true }));
+app.use(bodyParser.json({ limit: "50mb" }));
+app.use(bodyParser.urlencoded({ limit: "50mb", extended: true }));
 
 app.use(express.static(path.join(__dirname, "client/build")));
 
 
-app.post('/api/loadUserSettings', (req, res) => {
 
-	let connection = mysql.createConnection(config);
-	let userID = req.body.userID;
+app.post("/api/loadUserSettings", (req, res) => {
+  let connection = mysql.createConnection(config);
+  let userID = req.body.userID;
 
-	let sql = `SELECT mode FROM user WHERE userID = ?`;
-	console.log(sql);
-	let data = [userID];
-	console.log(data);
+  let sql = `SELECT mode FROM user WHERE userID = ?`;
+  console.log(sql);
+  let data = [userID];
+  console.log(data);
 
-	connection.query(sql, data, (error, results, fields) => {
-		if (error) {
-			return console.error(error.message);
-		}
+  connection.query(sql, data, (error, results, fields) => {
+    if (error) {
+      return console.error(error.message);
+    }
 
-		let string = JSON.stringify(results);
-		//let obj = JSON.parse(string);
-		res.send({ express: string });
-	});
-	connection.end();
+    let string = JSON.stringify(results);
+    //let obj = JSON.parse(string);
+    res.send({ express: string });
+  });
+  connection.end();
 });
 
+app.post("/api/getRequests", (req, res) => {
+  let connection = mysql.createConnection(config);
+  let sql = `SELECT * FROM requested_trips Where users_user_id = 1;`;
+  let data = [];
+  connection.query(sql, data, (error, results, fields) => {
+    if (error) {
+      return console.error(error.message);
+    }
+    let string = JSON.stringify(results);
+    let obj = JSON.parse(string);
+    res.send({ express: string });
+  });
+  connection.end();
+});
+
+app.delete("/api/deleteRequest", (req, res) => {
+  let connection = mysql.createConnection(config);
+  const postedTrip_id = req.body.postedTrip_id;
+  let sql = `DELETE FROM requested_trips WHERE (postedtrips_id = ?);`;
+  let data = [postedTrip_id];
+  connection.query(sql, data, (error, results, fields) => {
+    if (error) {
+      return console.error(error.message);
+    }
+    let string = JSON.stringify(results);
+    let obj = JSON.parse(string);
+    res.send({ express: string });
+  });
+  connection.end();
+});
 
+app.post("/api/postRequest", (req, res) => {
+  let connection = mysql.createConnection(config);
+  const pickupLocation = req.body.pickupLocation;
+  const dropoffLocation = req.body.dropoffLocation;
+  const deaprtureDate = req.body.deaprtureDate;
```

Comment:
Nothing major but departure is misspelt here - dont want this to lead to any issues in the future

Reply:
Fixed